api/src/test/java/com/epam/pipeline/manager/pipeline/PipelineRunManagerLaunchTest.java

```diff
@@ -142,13 +143,14 @@
     public void setUp() throws Exception {
         MockitoAnnotations.initMocks(this);
 
-        mockCloudRegionManager();
-        mockInstanceOfferManager();
-        mockPreferenceManager();
-        mockPipelineConfigurationManager();
-        mockToolManager();
-        mockParentRun();
+        mockCloudRegions();
+        mockPreferences();
```

Comment:
I think it's ok to inline these grouping mock methods. It'll show exactly what we mock in the test class right in place.

Reply:
Done!